### ISSUE TITLE
Widget Doesn't Start Playing Media Unless Open App Manually

### ISSUE THREAD

**Issue by GithubContributor832** [SimpleMobileTools/Simple-Music-Player#371]
If the app is closed and I hit play on widget it won't play media. It attempts to open with notification but fails going all black. It also doesnt update queue automatically or manually with no queue without recreating widget. Then lastly with no queue artist and song title is not there like in widget preview.  Picture below and using 5.9.0 release.

![Screenshot_20220106-035319_Shade_Launcher](https://user-images.githubusercontent.com/92182796/148356661-f56e2c28-417d-4a30-a5c5-17b5968a9cad.png)

**Comment by tibbi** [comment]
does it always happen? Or are there any exact steps for reproduction?

**Comment by GithubContributor832** [comment]
I figured this was relavent enough so I'm going to post here. The player notification pops up every app open before I hit play with or without a queue. Typically this should only happen when I play a queue or pause it with my initiation by starting the queue. Most music players I've used do it this way of manually. Call it a feature request or bug it just makes things more tidy in my opinion.
